ingest/sqs/pools/transformer/orderbook.go

```diff
@@ -69,12 +69,24 @@ func (pi *poolTransformer) updateOrderbookInfo(
 	nextBidTickIndex := tickIndexById(ticks, orderbook.NextBidTick)
 	nextAskTickIndex := tickIndexById(ticks, orderbook.NextAskTick)
 
+	bidAmountToExhaustAskLiquidity, err := sqscosmwasmpool.CalcAmountInToExhaustOrderbookLiquidity(sqscosmwasmpool.BID, nextAskTickIndex, ticks)
+	if err != nil {
+		return fmt.Errorf("error calculating bid amount to exhaust ask liquidity: %w", err)
+	}
+
+	askAmountToExhaustBidLiquidity, err := sqscosmwasmpool.CalcAmountInToExhaustOrderbookLiquidity(sqscosmwasmpool.ASK, nextBidTickIndex, ticks)
```

Comment:
Why are we starting from the next opposing ticks?

Based on the implementation of `CalcAmountInToExhaustOrderbookLiquidity`, the `startingIndex` is inclusive which doesn't feel right

Reply:
Because bid order is filled by ask liquidity and vice-versa, so we need to find the starting index that available and iterate through all of them to find how big the order is that required for exhausting liquidity on the other side of the orderbook.

Reply:
Got it - that makes sense